app/assets/stylesheets/landing.scss

```diff
@@ -63,6 +64,45 @@
   }
 }
 
+.footer-section {
+  align-items: center;
+  background-color: $orange;
+  box-shadow: 0 -8px 20px 0 #11111129;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  min-height: 20vh;
```

Comment:
let this addapt to its content, use padding to separate the content from the borders of the container